package.json

```diff
@@ -127,6 +127,7 @@
     "ws": "7.0.1",
     "xml2js": "0.4.19",
     "yaml": "1.6.0",
+    "yamljs": "0.3.0",
```

Comment:
what's the difference between `yaml` and `yamljs`? do we need both or can we get rid of one of them?

Reply:
"yaml" is a lib that allows you to convert YAML files to some kind of YAML AST. 
The only place we use it is in yaml_tools to split documents.

"yamljs" is a lib that allows us to convert a YAML file to a similar formatted javascript object tree (natural conversion), it is used in yaml_utils. I use it to parse and stringify YAML files so I could work with them in javascript.